marathon/tests/test_integration.py

```diff
@@ -0,0 +1,18 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+from copy import deepcopy
+
+from . import common
+
+pytestmark = pytest.mark.integration
+
+
+@pytest.mark.usefixtures("dd_environment")
+def test_check(aggregator, check):
+    check.check(deepcopy(common.INSTANCE_INTEGRATION))
```

Comment:
You can remove the fixture, and pass dd_environment as a argument, then you can do check.check(dd_environment).